src/Cli/TerminusCommands.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace UpdateTool\Cli;
+
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+use Robo\Common\ConfigAwareTrait;
+use Robo\Contract\ConfigAwareInterface;
+use Hubph\HubphAPI;
+use Hubph\Git\WorkingCopy;
+use Hubph\Git\Remote;
+
+class TerminusCommands extends \Robo\Tasks implements ConfigAwareInterface, LoggerAwareInterface
+{
+    use ConfigAwareTrait;
+    use LoggerAwareTrait;
+
+    /**
+     * Update terminus commands and releases in documentation repo.
+     *
+     * @command terminus:update-docs
+     */
+    public function terminusUpdateDocs($options = [
+        'as' => 'default',
+        'update-commands' => true,
+        'update-releases' => true,
+        'terminus-release' => 'latest',
+        'github-repo' => 'pantheon-systems/documentation',
+        'terminus-repo' => 'pantheon-systems/terminus',
+        'base-branch' => 'main',
+        'branch-name-prefix' => 'docs-update-terminus-',
+        'commit-message' => 'Update terminus information.',
+        'pr-body' => '',
+        'pr-title' => '[UpdateTool - Terminus Information] Update commands and releases.',
+        'dry-run' => false,
+    ])
+    {
+        $api = $this->api($options['as']);
+        $updateCommands = $options['update-commands'];
+        $updateReleases = $options['update-releases'];
+        if (!$updateCommands && !$updateReleases) {
+            throw new \Exception("Either commands or releases must be updated.");
+        }
+
+        $terminusRepo = $options['terminus-repo'];
+        $terminusRelease = $options['terminus-release'];
+
+        $terminusDir = sys_get_temp_dir() . '/update-tool/terminus/';
+        if (!is_dir($terminusDir)) {
+            mkdir($terminusDir, 0755, true);
+        }
+
+        if ($terminusRelease === 'latest') {
+            $terminusRelease = $this->getLatestRelease($api, $terminusRepo);
+        }
+        $downloadUrl = 'https://github.com/pantheon-systems/terminus/releases/download/' . $terminusRelease . '/terminus.phar';
+
+        file_put_contents($terminusDir . 'terminus.phar', file_get_contents($downloadUrl));
+        if (!file_exists($terminusDir . 'terminus.phar')) {
+            throw new \Exception("Failed to download terminus.phar from release $terminusRelease");
```

Comment:
Better to use `sprintf('Failed to download terminus.phar from release %s', $terminusRelease)`?